docs/WindowsBuild.md

```diff
@@ -1,408 +1,78 @@
+> If Git and Python 3.7+ are already installed on your system, you don’t need to reinstall them here (i.e ignore the first 2 commands).
 
-```cmd
-cmake -B S:\b\6 ^
-  -D BUILD_SHARED_LIBS=YES ^
-  -D CMAKE_BUILD_TYPE=RelWithDebInfo ^
-  -D CMAKE_INSTALL_PREFIX=C:\Library\Developer\Toolchains\unknown-Asserts-development.xctoolchain\usr ^
-  -D CMAKE_CXX_COMPILER=S:/b/1/bin/clang-cl.exe ^
-  -D CMAKE_CXX_FLAGS="-Xclang -fno-split-cold-code" ^
-  -D CMAKE_MT=mt ^
-  -D CMAKE_Swift_COMPILER=S:/b/1/bin/swiftc.exe ^
-  -D LLBUILD_SUPPORT_BINDINGS=Swift ^
-  -D dispatch_DIR=S:\b\2\cmake\modules ^
-  -D Foundation_DIR=S:\b\3\cmake\modules ^
-  -D SQLite3_INCLUDE_DIR=S:\Library\sqlite-3.28.0\usr\include ^
-  -D SQLite3_LIBRARY=S:\Library\sqlite-3.28.0\usr\lib\sqlite3.lib ^
-  -G Ninja ^
-  -S S:\swift-llbuild
+## Windows configuration
 
-ninja -C S:\b\6
-```
+### Symbolic link creation permissions
 
-- Add llbuild to your path:
+Symbolic links are used in some of the swift repositories. You need to grant permission to your user to create symbolic links in order to properly clone the repositories:
 
-```cmd
-path S:\b\6\bin;%PATH%
-```
+Using `gpedit.msc` add your user to the list of trusted user in `Computer Configuration\Windows Settings\Security Settings\Local Policies\User Rights Assignment\Create Symbolic links`.
 
-## Build Yams
+In a PowerShell shell, run the following command to enable Git symbolic links and to disable automatic line endings conversion to support all the test suite:
 
-```cmd
-cmake -B S:\b\7 ^
-  -D BUILD_SHARED_LIBS=YES ^
-  -D CMAKE_BUILD_TYPE=Release ^
-  -D CMAKE_INSTALL_PREFIX=C:\Library\Developer\Toolchains\unknown-Asserts-development.xctoolchain\usr ^
-  -D CMAKE_MT=mt ^
-  -D CMAKE_Swift_COMPILER=S:/b/1/bin/swiftc.exe ^
-  -D dispatch_DIR=S:\b\2\cmake\modules ^
-  -D Foundation_DIR=S:\b\3\cmake\modules ^
-  -D XCTest_DIR=S:\b\4\cmake\modules ^
-  -G Ninja ^
-  -S S:\Yams
-
-ninja -C S:\b\7
+```powershell
+git config --global --add core.symlink true
+git config --global --add core.autocrlf false
 ```
 
-## Build swift-argument-parser
-
-```cmd
-cmake -B S:\b\8 ^
-  -D BUILD_SHARED_LIBS=YES ^
-  -D CMAKE_BUILD_TYPE=Release ^
-  -D CMAKE_INSTALL_PREFIX=C:\Library\Developer\Toolchains\unknown-Asserts-development.xctoolchain\usr ^
-  -D CMAKE_MT=mt ^
-  -D CMAKE_Swift_COMPILER=S:/b/1/bin/swiftc.exe ^
-  -D dispatch_DIR=S:\b\2\cmake\modules ^
-  -D Foundation_DIR=S:\b\3\cmake\modules ^
-  -D XCTest_DIR=S:\b\4\cmake\modules ^
-  -G Ninja ^
-  -S S:\swift-argument-parser
+> [!IMPORTANT]
+> Please make sure to consider the security risks listed [here](https://docs.microsoft.com/en-us/windows/security/threat-protection/security-policy-settings/create-symbolic-links).
 
-ninja -C S:\b\8
-```
+## Cloning the repositories
 
-## Build swift-driver
+- Open a regular (non admin, non Visual Studio) Powershell shell.
+- Create a directory named S at the root of the C:\ drive: `mkdir C:\S`.
+- Navigate to that directory: `cd C:\S`.
```

Comment:
I think that we should document the use of `subst` unless they have a physical `S` drive. That uncovers a number of path related issues.